### DISCUSSION TITLE
More recommend content needed

### DISCUSSION THREAD

**Discussion by sfc9982** [community/community#13215] [Other Feature Feedback, Questions, & Ideas] [answered]
## Need more relateded topics recommenation needed

Such as I am looking for C, C++ repos recently, recommend popular other C/C++ repos for user.

More info on my frequently seeing topics or repos than other.

Overall, it's a good design, I hope you keep it.

**Comment by jfullstackdev** [comment, marked as answer]
do you star as much C / C++ topics ? 

other than that,

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
